source/renderer/app/features/discreet-mode/replacers/discreetWalletAmount.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import BigNumber from 'bignumber.js';
+import { formattedWalletAmount } from '../../../utils/formatters';
+import type { DiscreetValueReplacer } from '../types';
+
+export type DiscreetWalletAmountProps = {
+  amount: BigNumber,
+  currency?: string,
+  withCurrency?: boolean,
+  long?: boolean,
+};
+
+export const discreetWalletAmount: DiscreetValueReplacer = ({
+  amount,
+  withCurrency = true,
+  long = true,
+  currency = 'ADA',
```

Comment:
Not sure if its needs to be lower case. If yes pls use WalletUnits.ADA constant

Reply:
It has to be presented as upper case.